### DISCUSSION TITLE
The Budibase client library could not be loaded.

### DISCUSSION THREAD

**Discussion by vaskoZGU** [Budibase/budibase#12269] [Help] [answered]
Hello,
I have issue loading BB application after container image update last night.
![image](https://github.com/Budibase/budibase/assets/117993754/a5bafe44-7f8b-40a8-b318-d05ba290903a)

When I try to edit or manage the app I see below message:
![image](https://github.com/Budibase/budibase/assets/117993754/0608da6d-7a99-4ad6-bcc8-2ab0376837b2)

Can you please help me with this issue?

BB selfhosted on Synology docker
External MariaDB 

Thank you.

**Comment by ConorWebb96** [comment]
Hey @vaskoZGU,

- Does this happen on all your apps?
- Are you able to view the apps on different devices?
- Can you take some screenshots of your network requests and console logs for the page this happens on?

Additionally this thread might [help](https://github.com/Budibase/budibase/discussions/11502#discussioncomment-6700198).

Thanks in advance for the information.

**Reply by vaskoZGU** [reply]
Hi Conor,

Yes this happens on applications.
Tested on different devices and the issue is the same.
Attached you can find container log after container restart.
[_budibase_logs.txt](https://github.com/Budibase/budibase/files/13250325/_budibase_logs.txt)

Also export from BB: 
[b4e4fb784188402d863e9a9b9b9d14b1-1699018038060.log](https://github.com/Budibase/budibase/files/13250678/b4e4fb784188402d863e9a9b9b9d14b1-1699018038060.log)

I'm not sure what you mean by "screenshots of your network requests ". Can you tell me how to find what you need?
I'll check the thread mentioned.

Thank you and regards.

**Reply by vaskoZGU** [reply]
Maybe I found what you mean by network requests

![image](https://github.com/Budibase/budibase/assets/117993754/9ee8ce02-b462-48c4-be19-c84612df30c2)
![image](https://github.com/Budibase/budibase/assets/117993754/7d494c5c-26d2-4020-9675-8ba2797afbdf)
![image](https://github.com/Budibase/budibase/assets/117993754/4d0a89d3-dfb5-428f-a2c3-d9323765935e)

**Reply by jacobmaussocc** [reply]
Having the same issue after updating today.

**Reply by Hertizch** [reply]
Same issue here as well. My app is dead atm...

**Reply by vaskoZGU** [reply]
Is there any way to republish and/or export the app from command line?
I can't open the app to republish nor Manage section to export.

Tested so far with the same result:
- different browsers
- different devices
- removing cache and cookies
- browser incognito mode
- updated from 2.12.6 to 2.12.8 latest

**Reply by ConorWebb96** [reply, marked as answer]
Hey all this should be fixed once version 2.12.11 is released. Its currently in the process of being released right now.

Thanks for your patience.

**Reply by vaskoZGU** [reply]
Hello Conor,

Thank you for the quick fix. I confirm the issue is resolved with v.2.12.11.

Best Regards

**Reply by JayP718** [reply]
Hi @ConorWebb96,
I'm still receiving this issue in Kubernetes. I want to debug further on my end but I'm unsure into which of the service logs I should look.

It very well might be a networking  issue on my end but unsure where to look.

Thanks!

**Reply by s-okal** [reply]
I have also encountered this problem in Kubernetes. I have opened a separate discussion with greater detail.
